src/Codeception/Lib/Driver/Sqlite.php

```diff
@@ -22,19 +21,24 @@ public function __construct($dsn, $user, $password)
         parent::__construct($this->dsn, $user, $password);
     }
 
-    public function cleanup()
+    public function __destruct()
     {
-        $this->dbh = null;
         file_put_contents($this->filename, '');
-        $this->dbh = self::connect($this->dsn, $this->user, $this->password);
+    }
+
+    public function cleanup()
+    {
+        $this->dbh->exec('PRAGMA foreign_keys = OFF;');
+        $this->dbh->exec('PRAGMA writable_schema = 1;');
+        $this->dbh->exec('DELETE FROM sqlite_master WHERE type IN (\'table\', \'index\', \'trigger\');');
```

Comment:
Is that really needed? 

Probably Sqlite tests are failing because of this change: https://travis-ci.org/Codeception/Codeception/jobs/225994707#L2651